vtm-android-example/build.gradle

```diff
@@ -1,6 +1,7 @@
 apply plugin: 'com.android.application'
 
 dependencies {
+    compile project(':vtm')
```

Comment:
I would try with a fresh VTM clone and a clean Gradle cache to test if it's a local Gradle build problem.